tests/dotnet-sos/README.md

```diff
@@ -0,0 +1,52 @@
+### Debugging managed dotnet code
+
+## Prerequisites
+1. lldb:
+```
+sudo apt install lldb-10
+```
+2. dotnet-sos:
+```
+dotnet tool install -g dotnet-sos
+# Installs the SOS extension, and
+# updates .lldbinit file to load SOS on lldb startup
+dotnet-sos install
+```
+3. oelldb, SGX enclave extension for lldb:
+```
+git clone --branch working https://github.com/vtikoo/oelldb
```

Comment:
Is there a plan to merge oelldb into OE master or Mystikos's OE branch?

Reply:
The lldb extension is in OE master, but not in any release branch yet. Using the OE master version would also require mystikos to adopt the multi module debugger support in OE which @mikbras is already handling in his OE 0.16 rebase PR. Once thats in, maybe we can pull the lldb extension changes from master into Mystikos's OE branch.